src/V3Config.cpp

```diff
@@ -194,7 +197,7 @@ class V3ConfigModule {
         m_inline = true;
         m_inlineValue = set;
     }
-    void setPublic(bool set) { m_public = set; }
+    void addModulePragma(AstPragmaType pragma) { m_modPragmas.insert(pragma); }
```

Comment:
```suggestion
    void addModulePragma(const AstPragmaType& pragma) { m_modPragmas.insert(pragma); }
```

Reply:
Let me confirm.
AstPragmaType has an enum as a member variable. So pass-by-value is cheaper. That's why I did so.
If you prefer any Ast*  class to be pass-by-reference, that's also fine. I will apply the suggestion.

Reply:
Oh, sorry never mind on this one then.

Reply:
(One of those C++ sillynesses is that you need to know if a type is fundamental to know if the const& is appropriate!  Line tools will complain if you do it wrong - no const& on complex, or const& on non-complex.)